src/main/java/amazonreviewers/project/model/Comment.java

```diff
@@ -0,0 +1,110 @@
+package amazonreviewers.project.model;
+
+public class Comment {
```

Comment:
```suggestion
package amazonreviewers.project.model.dto;

public class CommentDto {
```
in fact, this is a DTO, later you will parse it to create Review (or Comment), Product and User entities